backend/src/libs/interfaces/repository.interface.ts

```diff
@@ -1,5 +1,5 @@
 interface IRepository<T = unknown> {
-  find(): Promise<T>;
+  find(id: unknown): Promise<T>;
```

Comment:
Are you sure this should be unknown?
In database schema all ids is integer.

Reply:
This is an interface for implementation. It should be with an unknown, right